### PR TITLE
implemented radio buttons for #1067

### LINKED ISSUE #1067: bullets for inc/azi
We could use a multiple-choice selection menu for inclination angle and azimuth and 'other' for a custom angle. Default should be backwards compatible. Suggestion by Josue.

### PR DESCRIPTION
**Pull request by Mathadon**
closes #1067 
default implementation is backwards compatible

### CODE REVIEW

**Comment by Mathadon** [comment]
@jotaborrajo in case you have feedback on this implementation?

**Comment by Mathadon** [comment]
unit tests require #1071

File: IDEAS/Buildings/Components/Interfaces/PartialSurface.mo [hunk 1]
```diff
@@ -48,24 +56,36 @@ partial model PartialSurface "Partial model for building envelope component"
...
+    energyDynamics=energyDynamics,
     linIntCon=linIntCon_a or sim.linearise,
-    A=A)
+    A=A,
+    final inc=incInt)
     "Multilayer component for simulating walls, windows and other surfaces"
     annotation (Placement(transformation(extent={{10,-10},{-10,10}})));
 
 protected
+  final parameter Modelica.SIunits.Angle aziInt=
```
Comment (jotaborrajo) [inline IDEAS/Buildings/Components/Interfaces/PartialSurface.mo]: I like this solution for the azimut and the inclination.

**Comment by Mathadon** [comment]
ok thanks, we'll proceed then!

**Pull request merged**